docs/knowledge_base/tips/long_callbacks/index.md

```diff
@@ -0,0 +1,73 @@
+# Long Running Callbacks
+
+Tasks in web applications can take a lot of time, which can lead to problems with communication between 
+the server (Python application) and the client (web browser).
+
+![Long Running Callbacks](long_running_callbacks.png){width=100%}
+
+To deal with this issue, Taipy has a helpful feature called "long-running callbacks" to address this problem. 
+These callbacks allow the server to do demanding tasks in the background while still making sure 
+the user interface stays quick to respond. 
+This article talks about the idea of long-running callbacks in Taipy, gives examples of how to use them, 
+and shows how they make the user experience better.
+
+## Example 1: Executing a Heavy Task in the Background
+
+Imagine a situation where a callback starts a task that requires a lot of resources and time to finish. 
+To make this work, we can use a straightforward approach:
+
+![Executing a Heavy Task in the Background](long_running_callbacks_2.png){width=100%}
+
+In the previous example, the Taipy function called `invoke_long_callback()` manages the resource-intensive task. 
+It sets up a separate background thread to run the `heavy_function()`, allowing the rest of the application to continue running. 
+The `on_action` function gets activated by a user action, like clicking a button, for instance.
+
+## Example 2: Monitoring Task Status
+
+Moreover, you can send notifications to the user's browser or update visual elements depending on the task's result. 
+Taipy offers a way to receive notifications when the task is complete, as shown below:
+
+![Monitoring Task Status](long_running_callbacks_3.png){width=100%}
+
+In this example, we introduce the `heavy_function_status()` function, which is invoked by the `invoke_long_callback()` method. 
+When the task is finished, this function is called. 
+This gives you the opportunity to provide the necessary notifications or make updates to the user interface depending 
+on whether the task was successful or not.
+
+## Handling heavy_function results: Updating the Application State
+
+To update the State according to the return of the `heavy_function`, you can modify the `heavy_function_status()` function as follows:
```

Comment:
"the return of the \`heavy_function\`, you can modify the \`heavy_function_status()\` function as follows:"
->
"the returned value from \*heavy_function()\*, you can modify \*heavy_function_status()\* as follows:"